src/index.html

```diff
@@ -11,7 +11,230 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="main-form"
+    >
+      <fieldset class="block-forms personal-block">
+        <legend class="title">
+          Personal information
+        </legend>
+        <div class="personal-info">
+          <label for="surname">
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              id="surname"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="personal-info">
+          <label for="name">
```

Comment:
You don't really need these additional wrappers
```suggestion
          <label for="name" class="personal-info">
```